tests/resources/test_resource_utils.py

```diff
@@ -0,0 +1,54 @@
+from unittest.mock import patch
+
+from gnomad.resources import resource_utils
+
+
+class TestTableResource:
+    @patch("hail.read_table")
+    def test_read_table(self, read_table):
+        resource = resource_utils.TableResource("gs://my-bucket/table.ht")
+
+        ds = resource.ht()
+        read_table.assert_called_with("gs://my-bucket/table.ht")
+        assert ds == read_table.return_value
+
+
+class TestMatrixTableResource:
+    @patch("hail.read_matrix_table")
+    def test_read_matrix_table(self, read_matrix_table):
+        resource = resource_utils.MatrixTableResource("gs://my-bucket/matrix_table.mt")
+
+        ds = resource.mt()
+        read_matrix_table.assert_called_with("gs://my-bucket/matrix_table.mt")
+        assert ds == read_matrix_table.return_value
+
+
+class TestPedigreeResource:
+    @patch("hail.Pedigree.read")
+    def test_read_pedigree(self, read_pedigree):
+        resource = resource_utils.PedigreeResource("gs://my-bucket/pedigree.ped")
+
+        ds = resource.pedigree()
+        read_pedigree.assert_called()
```

Comment:
Is there a reason why this class uses `assert_called()` vs `assert_called_with()`?

Reply:
The PedigreeResource methods call `hl.Pedigree.read` and `hl.import_fam` with some additional arguments, but I was only interested in testing the first argument (the path) here.

`assert_called_with("path")` would fail because the function was called with another argument.